docs/src/markdown/index.md

```diff
@@ -309,3 +309,21 @@ jobs:
 
 In *this* project, we actually use `tox` to make running our tests locally and in CI easier. If you would like to use
 `tox` as well, you can check out how this project does it by taking a look at the source.
+
+## Usage as `pre-commit` Hook
+
+PySpelling can be used as [`pre-commit`](https://pre-commit.com/) hook. To use it as `pre-commit` hook, please have a
```

Comment:
One small nit "can be used as _a_ `pre-commit` hook". Or maybe "can be used _in_ `pre-commit` _hooks_". Emphasis added only for illustration.